numpy/core/_add_newdocs.py

```diff
@@ -1454,11 +1454,10 @@ def luf(lamdaexpr, *args, **kwargs):
     Values are generated within the half-open interval ``[start, stop)``
     (in other words, the interval including `start` but excluding `stop`).
     For integer arguments the function is equivalent to the Python built-in
-    `range <https://docs.python.org/library/functions.html#func-range>`_ function,
-    but returns an ndarray rather than a list.
+    `range` function, but returns an ndarray rather than a list.
 
     When using a non-integer step, such as 0.1, the results will often not
-    be consistent.  It is better to use ``linspace`` for these cases.
+    be consistent.  It is better to use `numpy.linspace` for these cases.
```

Comment:
Can we use `np` yet?

Reply:
sadly, no. PR #11344 was closed without a fix. Se also PR #11377 which tried to allow the monkeypatching needed to make `np` work.